core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -421,7 +421,7 @@ export class DotEditContentHtmlService {
     private setMaterialIcons(): void {
         const doc = this.getEditPageDocument();
         const link = this.dotDOMHtmlUtilService.createLinkElement(
-            'dotAdmin/assets/material-icons.css'
+            '/dotAdmin/assets/material-icons.css'
         );
```

Comment:
Create a variable for this, and can we test this `path`? That way, we avoid having the same `issue` if someone accidentally changes it.

Reply:
This service has all the test skipped, so I'm not really sure about the testing, but I can add the constant